utils/results/conversion/sarifparser/sarifparser.go

```diff
@@ -478,9 +483,10 @@ func parseScaToSarifFormat(params scaParseParams) (sarifResults []*sarif.Result,
 	return
 }
 
-func getScaIssueSarifRule(ruleId, ruleDescription, maxCveScore, summary, markdownDescription string) *sarif.ReportingDescriptor {
+func getScaIssueSarifRule(impactPaths [][]formats.ComponentRow, ruleId, ruleDescription, maxCveScore, summary, markdownDescription string) *sarif.ReportingDescriptor {
 	cveRuleProperties := sarif.NewPropertyBag()
 	cveRuleProperties.Add(severityutils.SarifSeverityRuleProperty, maxCveScore)
+	cveRuleProperties.Add(sarifutils.SarifImpactPathsRulePropertyKey, impactPaths)
```

Comment:
```suggestion
        if len(impactPaths) > 0 {
         	cveRuleProperties.Add(sarifutils.SarifImpactPathsRulePropertyKey, impactPaths)
        }
```
 I would add a check to see if not empty, only adding if there is something to add